test/integration/glob.spec.js

```diff
@@ -179,7 +179,7 @@ function execMochaWith(validate) {
       node +
         ' "' +
         path.join('..', '..', '..', '..', 'bin', 'mocha') +
-        '" -R json-stream ' +
+        '" -R json-stream --config /dev/null ' +
```

Comment:
I can't remember why I had to do this.

Reply:
But would be nice to know...

Reply:
- [ ] see if I can't revert this; if I can't, then explain why

Reply:
The reason is that our config requires `test/setup.js`, relative from the root, but the cwd used is `test/integration/fixtures/glob`, so `test/setup.js` won't be found.